ultraplot/tests/test_axes.py

```diff
@@ -187,3 +187,22 @@ def test_subset_format():
     # Shorter than number of axs
     with pytest.raises(ValueError):
         axs.format(title=["a"])
+
+
+def test_unsharing():
+    """
+    Test some basic properties of unsharing axes.
+    """
+    fig, ax = uplt.subplots(ncols=2)
+    # Does nothing since key is not an axis or a view
+    ax[0]._unshare(which="key does not exist")
```

Comment:
Should thjis raise on error instead of silently failing?

Reply:
We can raise a warning

Reply:
You think we need a test?

Reply:
Yes we should test.